test/functionalTest/cases/1985_empty_entity_type/empty_entity_type_in_subscription.test

```diff
@@ -0,0 +1,78 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Empty entity type in subscription
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Attempt to create a subscription with an empty entity type, see error
```

Comment:
To make this complet and align with

"Fix: if entity::type is given but as an empty string, in payload of POST /v2/subscriptions or **PATCH /v2/subscriptions/{sub}**, an error is returned (#1985)"

I'd suggest to add

02. Successfull creation of a subscription
03. Attemp to update the existing subscription with an empty entity type, see error